py-base/environment.yml

```diff
@@ -79,18 +86,26 @@ dependencies:
   - regionmask
   - requests
   - retrying
-  - rise
+  - rioxarray
+  # 2/12/23: Caused error during mamba env create
+  # - rise
   - s3fs
   - scipy
   - seaborn
   - searvey
   - seawater
+  # 2/15/2023: Heads-up: shapely 2.0 may be causing havoc,
+  # and it may be a good idea to pin it to <2
   - shapely
   - siphon
   - tensorflow-cpu
   - unzip
   - xarray
   - xesmf
   - xgcm
-  - xlrd
+  # - xlrd  # deprecated, use openpyxl instead
   - zarr
+  # - pip:
+    # - echopy
```

Comment:
This is in conda now. You can try it out here.

Reply:
Thank you!!!